package.json

```diff
@@ -31,7 +32,8 @@
         "format": "yarn d2-style apply",
         "lint": "yarn d2-style check",
         "e2e": "yarn playwright test --ui",
-        "generate-types": "rm -rf ./src/types/generated && yarn openapi --input https://dev.api.im.dhis2.org/swagger.yaml --output ./src/types/generated --client axios --useOptions --useUnionTypes --exportCore false --exportServices false --exportModels true --exportSchemas false && yarn prettier src/types/generated --write --loglevel silent"
+        "generate-types": "rm -rf ./src/types/generated && yarn openapi --input https://dev.api.im.dhis2.org/swagger.yaml --output ./src/types/generated --client axios --useOptions --useUnionTypes --exportCore false --exportServices false --exportModels true --exportSchemas false && yarn prettier src/types/generated --write --loglevel silent",
+        "prepare": "husky"
```

Comment:
See unresolved comment above (not sure why it's not showing in the code-tab anymore)

Reply:
@Mohammer5  @tonsV2  I'll be closing this pr as I had some issues while rebasing this branch... the implementation with most comments resolved can be found in this [link](https://github.com/dhis2-sre/im-web-client/pull/519)